bin/GameIndex.yaml

```diff
@@ -26912,7 +26913,7 @@ SLPM-66813:
   name: "Sangokushi IX [Koei Selection]"
   region: "NTSC-J"
 SLPM-66814:
-  name: "Pure x Cure Re-covery"
+  name: "Pure x Cure Recovery"
```

Comment:
It's actually Re:covery see very small box art https://psxdatacenter.com/psx2/images2/covers/SLPM-66814.jpg

Reply:
They do like doing stuff like that. Lemme do quick fix.